tests/chapter1-exercise-caching-tests.md

```diff
@@ -114,3 +114,43 @@ The responses cached earlier should still be valid.
     >>> body = browser.URL("http://wbemocks.test/cache_me2").request()
     >>> body
     'Keep this for a while, also'
+    
+    
+ Test that caches aren't used if scheme, host, or port are changed
+ Add scheme, host, or port to your cache key
```

Comment:
This should be something students could read, not sorta disjointed sentence fragments.